celery_app/celery.py

```diff
@@ -0,0 +1,29 @@
+from celery import Celery
+
+# broker='redis://localhost:6379/0',
+#                     backend='db+sqlite:///results.sqlite',
+#                     include=['celery_app.tasks']
+
+celery_app = Celery('celery_app')
+
+celery_beat_schedule = {
+    "call-every-10": {
+        "task": "celery_app.tasks.calculate",
+        "schedule": 4,
+        "args": (3,),
+    }
+}
+# Optional configuration, see the application user guide.
+celery_app.conf.update(
+    broker_url="redis://localhost:6379/0",
+    result_backend="db+sqlite:///results.sqlite",
```

Comment:
same here lets use redis for broker and result backend

Reply:
The `celery_app` package was not supposed to be included. I was using it for testing and forgot to remove it.